src/client/pages/Admin/Admin.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import { ClientRoutes } from '@meta/app'
+import { Users } from '@meta/user'
+
+import { useUser } from '@client/store/user'
+
+const Admin: React.FC = () => {
+  const user = useUser()
+
+  const isAdministrator = Users.isAdministrator(user)
+
+  if (!isAdministrator) window.location.href = ClientRoutes.Root.path
```

Comment:
is it better to use react-router hooks

Reply:
yes, try using react router in this case

Reply:
there are two solutions:

```
        <Route
          path={`${ClientRoutes.Admin.Root.path.absolute}/*`}
          element={isAdministrator ? <Admin /> : <Navigate replace to={ClientRoutes.Root.path} />}
        />
```

OR

```
const navigate = useNavigate()
if (!isAdministrator) navigate(ClientRoutes.Root.path)
```